advanced.md

```diff
@@ -167,4 +167,112 @@ public void toggleAllDigitalPins( bool setPinsHigh )
 		firmata.sendDigitalPort( 1, 0 );	//all pins low
```

Comment:
I understand the issues with magic numbers, and typically I would agree. I don't agree in this case, as this is just a simple example of advanced usage of the Firmata 'sendPort'. This value '0' as it is currently written is further explained in the comment, is explained in great detail in the section above, and is one of four total usages in the code sample.

Nevertheless, I propose '0' being changed to '0x00' to help illustrate the fact that it is a binary representation of the state of 8 pins, not a magic value.